lib/Template/gramma.peg

```diff
@@ -70,12 +70,12 @@ ValuelessAttribute = name:AttributeName {
     };
 }
 
-AttributeName = [a-zA-Z0-9\-]+ { return text() }
+AttributeName = '#'? [a-zA-Z0-9\-]+ { return text() }
```

Comment:
i suppose it's like [pragma in c#](https://msdn.microsoft.com/en-us/library/x74w198a.aspx). it's gives special instructions for compiler